codebuild/bin/install_s2n_head.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+set -ex
+pushd "$(pwd)"
+
+usage() {
+    echo "install_s2n_head.sh build_dir"
+    exit 1
+}
+
+if [ "$#" -ne "1" ]; then
+    usage
+fi
+
+BUILD_DIR=$1
+source codebuild/bin/jobs.sh
+cd "$BUILD_DIR"
+
+mkdir s2n_head
+cd s2n_head
+
+# Clone the most recent s2n commit
+git clone https://github.com/aws/s2n-tls
+
+mkdir ../build
+cd ../build
+
+cmake ../s2n_head/s2n-tls -DCMAKE_PREFIX_PATH="$BASE_S2N_DIR"/test-deps/openssl-1.1.1/
+make -j $JOBS
+
+# Create symbolic link for s2nc and s2nd
+ln -sf "$BUILD_DIR"/build/bin/s2nc "$BASE_S2N_DIR"/bin/s2nc_head
+ln -sf "$BUILD_DIR"/build/bin/s2nd "$BASE_S2N_DIR"/bin/s2nd_head
```

Comment:
Would it be better to copy it out rather than link? Especially in the cases where you're building into a temp dir it's hard to assume anything about persistence.